src/hello/components/Hello.tsx

```diff
@@ -1,12 +1,10 @@
-// @flow
-
-import React, {Component} from 'react'
-import {observer} from 'mobx-react'
-import css from './Hello.css'
+import * as React from 'react'
+import { observer } from 'mobx-react'
 import helloStore from '../store'
+import * as css from './Hello.css'
 
 @observer
-export class Hello extends Component {
+export class Hello extends React.Component {
```

Comment:
I think a good convention would be for each react component to specify an `interface Props {}` and `interface State {}`, and then `export class C extends React.Component<Props, State>`